test/cppsuite/tests/test_live_restore.cpp

```diff
@@ -323,11 +347,20 @@ main(int argc, char *argv[])
     }
     auto crud_session = connection_manager::instance().create_session();
 
-    if (!fresh_start) {
+    if (!fresh_start && !background_debug) {
         configure_database(crud_session);
     }
 
-    do_random_crud(crud_session, fresh_start);
+    if (background_debug) {
```

Comment:
2 requests here:
- I don't see any tests that call `test_list_restore.cpp` with the -f flag. Can we add it to every nth iteration in the shell script?
- It'd also be nice to have a python test for running background thread until it completes. I'm looking at the cppsuite test as stress testing and python as functional testing

Reply:
Did you mean the -b flag? I can add a python test but I think we should not execute it in a shell script at this stage until we get better options parsing.

Reply:
See: https://jira.mongodb.org/browse/WT-13889

Reply:
Yep, should've said the `-b` flag. 
I'm happy to skip the ccpsuite test if we have a python test where background migration runs to completion

Reply:
Data validation: dfde65f67